src/App.js

```diff
@@ -14,11 +14,59 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Selected good: -</h1>
-    {goodsFromServer.length}
-  </div>
-);
+class App extends React.Component {
+  state = {
+    selectedGood: 'Jam',
+  }
+
+  setSelectedGood = good => this.setState(() => ({
+    selectedGood: good,
+  }))
+
+  render() {
+    return (
+      <div className="App">
+        <ul>
+          {goodsFromServer.map(good => (
+            <>
+              <li
+                key={good}
+                className={this.state.selectedGood === good ? 'active' : ''}
+              >
+                {`${good} `}
+                <button
+                  type="button"
+                  hidden={this.state.selectedGood === good}
```

Comment:
використай conditional rendering
```suggestion
                className={this.state.selectedGood === good ? 'active' : ''}
              >
                {good} 
                {this.state.selectedGood === good && (
                  <button />
                )}
```

Reply:
done